apps/docs/pages/guides/getting-started/tutorials/with-sveltekit.mdx

```diff
@@ -69,249 +69,380 @@ Install the auth helpers for SvelteKit:
 npm install @supabase/auth-helpers-sveltekit
 ```
 
-Update your `src/routes/+layout.svelte`:
-
-```html title=src/routes/+layout.svelte
-<script lang="ts">
-  import { supabase } from '$lib/supabaseClient'
-  import { invalidate } from '$app/navigation'
-  import { onMount } from 'svelte'
-  import './styles.css'
+Add the code below to your `src/hooks.server.ts` to initialize the client on the server:
 
-  onMount(() => {
+```ts title=src/hooks.server.ts
+// src/hooks.server.ts
+import {
+  PUBLIC_SUPABASE_URL,
+  PUBLIC_SUPABASE_ANON_KEY
+} from '$env/static/public';
+import { createSupabaseServerClient } from '@supabase/auth-helpers-sveltekit';
+import type { Handle } from '@sveltejs/kit';
+
+export const handle: Handle = async ({ event, resolve }) => {
+  event.locals.supabase = createSupabaseServerClient({
+    supabaseUrl: PUBLIC_SUPABASE_URL,
+    supabaseKey: PUBLIC_SUPABASE_ANON_KEY,
+    event
+  });
+
+  /**
+   * a little helper that is written for convenience so that instead
```

Comment:
A convenience helper so we can just call call `await getSession()`  instead `const { data: { session } } = await supabase.auth.getSession()`